client/src/components/OverviewComponent/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button } from 'antd';
+
+import './style.css';
+
+const OverviewComponent = ({
+  firstTitle,
+  secondTitle,
+  content,
+  buttonText,
+  imageSource,
+  btnLink,
+  isRight,
```

Comment:
@AlaaSaadeddin I think this could use a more descriptive name, like `showPhoto`.  Actually, maybe you don't need it at all, and can just check if `imageSource` is not falsy when deciding to show the container-photo div or not.

Reply:
well i named it isRight because there is 3 pages as a child for this common component
so just one of these pages its photo is on the left , just to decide direction photo
relates #113

Reply:
Ah, got it.  Thanks for explaining!